src/common/NavigationProvider.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+import { NavigationContainer, Theme } from '@react-navigation/native';
+import { useTheme } from 'react-native-paper';
+
+interface Props {
+  children?: React.ReactNode;
+}
+
+export function NavigationProvider({ children }: Props) {
+  const theme = useTheme<Theme>();
+
+  return <NavigationContainer theme={theme}>{children}</NavigationContainer>;
+}
```

Comment:
This new `NavigationProvider` wraps the standard provider `NavigationContainer` so that we can access the theme context from Paper and use the same theme for React Navigation.

Reply:
I'd second that notion.  It might be forward thinking to go with `BrandedNavigationContainer` so we can utilize as much of the BrandConfigProvider as needed (possibly beyond just `useTheme`?).  Could be over-thinking it and `ThemedNavigationContainer` could be the way to go.  No strong pref on `Container` vs. `Provider` suffix, I'd just default to the former to match their approach.

Reply:
Yep, that will be better.  Renamed to `ThemedNavigationContainer`